Test/Unit/Block/FormTest.php

```diff
@@ -0,0 +1,154 @@
+    private $contextMock;
+
+    /**
+     * @var BlockForm
+     */
+    private $block;
+
+    /**
+     * @var Config
+     */
+    private $configHelperMock;
+
+    /**
+     * @var SessionManager
+     */
+    private $sessionManager;
+
+    /**
+     * @var BackendQuote
+     */
+    private $quoteMock;
+
+    /**
+     * @var CustomerCreditCardCollectionFactory
+     */
+    private $customerCreditCardCollectionFactoryMock;
+
+    /**
+     * @var Address
+     */
+    private $addressMock;
+
+    /**
+     * @inheritdoc
+     */
+    protected function setUp()
+    {
+        $this->initRequiredMocks();
+        $this->initCurrentMock();
+    }
+
+    private function initRequiredMocks()
+    {
+        $this->configHelperMock = $this->createMock(Config::class);
+        $this->contextMock = $this->createMock(Context::class);
+
+        $this->quoteMock = $this->getMockBuilder(BackendQuote::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getBillingAddress'])
+            ->getMock();
+
+        $this->addressMock = $this->getMockBuilder(Address::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getCustomerId'])
+            ->getMock();
+
+        $this->sessionManager = $this->getMockBuilder(SessionManager::class)
+            ->disableOriginalConstructor()
+            ->setMethods(['getQuote'])
+            ->getMock();
+
+        $this->sessionManager->method('getQuote')->willReturn($this->quoteMock);
+        $this->customerCreditCardCollectionFactoryMock = $this->getMockBuilder(CustomerCreditCardCollectionFactory::class)
+            ->setMethods(['create', 'getCreditCardInfosByCustomerId'])
+            ->getMock();
+    }
+
+    private function initCurrentMock()
+    {
+        $this->block = $this->getMockBuilder(BlockForm::class)
+            ->setMethods(['getQuoteData'])
+            ->setConstructorArgs(
+                [
+                    $this->contextMock,
+                    $this->configHelperMock,
+                    $this->sessionManager,
+                    $this->customerCreditCardCollectionFactoryMock
+                ]
+            )
+            ->getMock();
+    }
+
+    /**
+     * @test
+     */
+    public function testGetCustomerCreditCardInfo_withCustomer()
+    {
+        $this->addressMock->expects(self::once())->method('getCustomerId')->willReturn(11111);
+        $this->quoteMock->expects(self::once())->method('getBillingAddress')->willReturn($this->addressMock);
+        $this->block->expects(self::once())->method('getQuoteData')->willReturn($this->quoteMock);
+
+        $this->customerCreditCardCollectionFactoryMock->expects(static::once())->method('create')->willReturnSelf();
+        $this->customerCreditCardCollectionFactoryMock->expects(static::once())
+            ->method('getCreditCardInfosByCustomerId')
+            ->withAnyParameters()
+            ->willReturnSelf();
+
+        $this->block->getCustomerCreditCardInfo();
```

Comment:
What does this test check?

Reply:
@YevhenShBolt This test is to check and ensure that the method getCreditCardInfosByCustomerId gets called once when the method getCustomerCreditCardInfo is called.

Reply:
@YevhenShBolt  I updated the code in this PR https://github.com/BoltApp/bolt-magento2/pull/478. Can you check?